superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -754,8 +754,9 @@ def generate_newer_platform_components(basepath, distro):
                 newer_sys_comps_file.write(txt_output)
                 ok('Wrote {0}'.format(newer_sys_comps_path))
         except (OSError, RuntimeError) as e:
-            err('Failed to write change summary {} to disk! {}'.format(
-                newer_sys_comps_path, e))
+            err('Failed to write newer-platform-components.list {0} '
+                'to disk! {1}'.format(
+                    newer_sys_comps_path, e))
```

Comment:
`Failed to write newer-platform-components.list {0}` -> `Failed to write {0}` (newer_sys_comps_path is the pathname).

Commit summary: `bitbake: Fix exception text in generate_newer_platform_components` -> `yocto_recipe.py: Fix exception text in generate_newer_platform_components()`